src/pages/Details/Detail.scss

```
@@ -0,0 +1,202 @@
+.detail {
+  width: 1351px;
+  margin: 0 auto;
+  background-color: #ffffff;
+  .contents {
+    width: 1018px;
+
+    margin: 0 150.5px;
+    padding: 0 0 120px;
+    background-color: #ffffff;
+    .DetailAreaPadding {
```

```suggestion
    .detailAreaPadding {
```
첫 글자가 대문자인 PascalCase는 컴포넌트 이름에만 적용이 되는 부분입니다!
아래도 이렇게 작성하신 부분이 많은데 전부 수정해주세요!